wg-lts/charter.md

```diff
@@ -0,0 +1,95 @@
+# WG Long Term Support
+
+This charter adheres to the conventions described in the [Kubernetes Charter README]
+and uses the Roles and Organization Management outlined in [sig-governance].
+
+[sig-governance]: https://github.com/kubernetes/community/blob/master/committee-steering/governance/sig-governance.md
+[Kubernetes Charter README]: https://github.com/kubernetes/community/blob/master/committee-steering/governance/README.md
+
+## Scope
+
+The Long Term Support Working Group (WG LTS) is organized with the goal of developing a better understanding what "Long Term Support" might mean for Kubernetes, those who support Kubernetes, and end users, as well as to investigate changes the Kubernetes project could make related to long term support. The working group will also determine feasibility, benefits, **costs**, and prerequisites of such changes.
+
+In the first phase of the working group, we will collect information related to the needs and wants regarding support periods from end-users and people supporting Kubernetes. This will be done by reaching out to users and cluster operators (e.g. via surveys), to better understand:
+
+* in-use versions (in the form of a user/vendor survey) and reasons for remaining on those versions
+* constraints on deployment and upgrade patterns / timelines (e.g. edge deployments, regulated industries, retail, etc)
+* expected/required support periods from users/vendors
+* support periods of core Kubernetes dependencies
+* support periods of other components required to run Kubernetes clusters (OS, network, storage, etc)
+* what users/vendors are currently doing to support releases past community support End Of Life (EOL)
+* what do users/vendors do with EOL clusters today? what would they do if EOL was extended by N months/years?
```

Comment:
> EOL was extended by N months/years.

Extending the current long-term support of all versions?

As I know, some other software will mark some versions as long support and others as not.
- for instance, we may mark one release of a year as long support for some extended time only. This may save some effort of the community.

Reply:
That isn't known yet, that's one of the questions to investigate/answer